modules/mobile/app/services/mobile/v0/legacy_disability_rating/proxy.rb

```diff
@@ -29,7 +29,6 @@ def get_disability_ratings # rubocop:disable Metrics/MethodLength
           raise Common::Exceptions::BackendServiceException, 'MOBL_502_upstream_error'
         rescue => e
           if e.respond_to?('response')
-            Rails.logger.info('LEGACY DR ERRORS WITH RESPONSE', error: e)
```

Comment:
both of these logs are followed by an error so worst case we can look up the error